api/include/opentelemetry/trace/propagation/HttpTraceContext.cc

```diff
@@ -0,0 +1,129 @@
+#include <vector>
+
```

Comment:
I think you want to wrap in OPENTELEMETRY_BEGIN_NAMESPACE ... OPENTELEMETRY_END_NAMESPACE